src/components/Footer/footer.module.css

```diff
@@ -0,0 +1,69 @@
+.footer {
+    background: #27282c;
+}
+
+.inner {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    padding: 32px;
+}
+
+.nav {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+.navitem {
+    display: flex;
+    align-items: center;
+}
+
+.navitem::after {
+    display: inline-block;
+    content: '';
+    border-right: 1px solid rgba(255, 255, 255, 0.5);
+    height: .75em;
+    margin: 0 1ex;
+    width: 0;
+}
+
+.navitem:last-of-type::after {
+    display: none;
+}
+
+.link {
+    color: rgba(255, 255, 255, 0.7);
+    text-decoration: none;
+    border-bottom: 0 none transparent;
+}
+
+.support {
+    color: rgba(255, 255, 255, 0.5);
+    display: flex;
+    align-items: center;
+}
+
+.supportlabel {
+    margin-right: 1ex;
```

Comment:
We usually don't set margins through ex, it looks weird for me.

Reply:
Do you have a reason for that? 🤔

Reply:
Yes, I assume that Denis usually sets the margins using px in the design layout, not ex. I this case, I see that the margin is 16px:
<img width="713" alt="Screenshot 2021-12-13 at 15 37 32" src="https://user-images.githubusercontent.com/3338311/145813971-3ec6eef1-6ed1-45ff-a4fd-164cd9529187.png">

Reply:
Ah, got it. Of course it is worth using ex if it is appropriate.